kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/RuntimeDataServiceIntegrationTest.java

```diff
@@ -900,10 +900,17 @@ public void testFindTaskEvents() throws Exception {
 
         try {
 
-            List<TaskSummary> tasks = taskClient.findTasks("yoda", 0, 10);
+            List<TaskSummary> tasks = taskClient.findTasks("yoda", 0, 50);
```

Comment:
@jakubschwan maybe instead of using findTasks we could use findTasksByStatusByProcessInstanceId (https://github.com/droolsjbpm/droolsjbpm-integration/blob/master/kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/UserTaskServicesClient.java#L119) instead so we could avoid this loop and increased number of records to load. Wdyt?